db/migrate/20240704062706_add_qid_to_questions.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
```

Comment:
@jaydip1235  looks like you are missing some migrations here,`PG::UndefinedTable: ERROR: relation "questions" does not exist`. I don't see the table questions anywhere in both the PRs, make sure all code is in this Pull request.